doc/content/getting-started/programming-software/st-examples/openocd/_index.md

```diff
@@ -27,3 +27,62 @@ Some parts of the command above are optional:
 openocd -f interface/stlink.cfg -f target/stm32wlx.cfg -c "program <elf-file-location> verify reset exit"
 ```
 The binary file is extracted from this file during compilation of the Generic Node firmware, so it should not make a difference which file you use.
+
+## Using VSCode
+
+OpenOCD can be used in combination with VSCode in order to flash and debug applications.
+
+1. Install [Cortex-Debug](https://marketplace.visualstudio.com/items?itemName=marus25.cortex-debug)
+2. Create `launch.json` inside `.vscode` folder and add the following lines
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug-STM32WL",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceRoot}/Software/build/debug/app/basic/basic.elf",
+            "args": [],
+            "stopAtEntry": true,
+            "cwd": "${workspaceRoot}",
+            "environment": [],
+            "externalConsole": false,
+            "debugServerArgs": "-f interface/stlink.cfg -f target/stm32wlx.cfg",
+            "serverLaunchTimeout": 20000,
+            "filterStderr": true,
+            "filterStdout": false,
+            "serverStarted": "Info\\ :\\ [\\w\\d\\.]*:\\ hardware",
+            "setupCommands": [
+                { "text": "cd ${workspaceRoot}/Software/build" },
+                { "text": "file debug/app/basic/basic.elf", "description": "load file", "ignoreFailures": false},
+                { "text": "target extended-remote localhost:3333", "description": "connect to target", "ignoreFailures": false },
+                { "text": "monitor reset halt", "description": "perform a reset and halt the target", "ignoreFailures": false },
+                { "text": "load", "description": "flash target", "ignoreFailures": false },
+                { "text": "monitor reset init", "description": "perform a reset and init the target", "ignoreFailures": false },
+                { "text": "set output-radix 16", "description": "set the default numeric base to 16", "ignoreFailures": false }
+            ],
+            "logging": {
+                "moduleLoad": true,
+                "trace": true,
+                "engineLogging": true,
+                "programOutput": true,
+                "exceptions": true
+            },
+            "MIMode": "gdb",
+            "miDebuggerPath": "/Users/UserName/opt/gcc-arm-none-eabi-6-2017-q2-update/bin/arm-none-eabi-gdb",
+            "debugServerPath": "/usr/local/bin/openocd"
+        }
+    ]
+}
+```
+> **Note:** You need to update "program" and "setupCommands" fields with the application you plan to flash and debug.
+
+> **Please also note:** You need to update "miDebuggerPath" and "debugServerPath" fields with the system paths of your tool chain gdb and openOCD binary.
```

Comment:
```suggestion
> **Note:** You need to update `program` and `setupCommands` fields with the application you plan to flash and debug.

> **Please also note:** You need to update `miDebuggerPath` and `debugServerPath` fields with the system paths of your tool chain gdb and openOCD binary.
```

Reply:
Is it **openOCD** or **OpenOCD**? I see both usages in this doc, so please make sure to be consistent there. Also, I'm pretty sure this repo has the ability to wrap up notes (https://github.com/TheThingsIndustries/generic-node-docs/blob/master/doc/themes/generic-node/layouts/shortcodes/note.html). So instead of using:

`> **Note:** ...`

you should use:

`{{< note >}} ... {{</ note >}}`